src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -247,7 +246,7 @@ typedef struct uct_rc_mlx5_iface_common {
     uct_ib_mlx5_cq_t                 cq[UCT_IB_DIR_NUM];
     struct {
         struct ibv_qp                *cmd_qp;    /* set if QP was created by UCX */
-        uct_rc_mlx5_cmd_wq_t         cmd_wq;
+        uct_rc_mlx5_cmd_qp_t         cmd_wq;
```

Comment:
should we rename the field cmd_wq to cmd_qp?
what about the existing field cmd_qp, isn't 'struct ibv_qp*' already part of uct_rc_mlx5_cmd_qp_t::super::super ?